Verify content entity name does contain the entity type (#1300)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

> * Added a validation to ensure that the integration name, integration display name, script name, and playbook name will not contain the entity type.

Reviewer: ```suggestion
* Added validation to ensure that integrations, scripts, and playbooks do not contain the entity type in their names.
```
